assets/scripts/src/choices.js

```diff
@@ -2198,7 +2202,16 @@ class Choices {
       // Highlight given option, and set accessiblity attributes
       passedEl.classList.add(this.config.classNames.highlightedState);
       passedEl.setAttribute('aria-selected', 'true');
-      this.containerOuter.setAttribute('aria-activedescendant', passedEl.id);
+
+      const hasActiveDropdown = this.dropdown.classList.contains(
+        this.config.classNames.activeState
+      );
+      if (hasActiveDropdown) {
```

Comment:
That is something I was wondering about as well, it would be nice to add the links as soon as dropdown is open. This way the currently highlighted option will be read immediately.

`_highlightChoice` seemed like a good place to update the links because it ensures that they stay up-to-date with mouse hover/arrow key press events.

I've also tried to add a call to `_highlightChoice` at the end of `showDropdown`, which updates the links properly, but prevents virtual keyboard from appearing in IE11. However, the keyboard appears as expected if we call `_highlightChoice` on a next tick (e.g., a timeout of 0). That approach ensures that links are added as soon as dropdown is open and keeps the virtual keyboard in IE11, but seems a bit hacky. I am wondering if there might be a better place to call `_highlightChoice` or set the links when the dropdown is open.

Let me know what are your thoughts on that.